hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -41,6 +41,27 @@
            Randomly select key size to verify all supported key sizes are working'''
       milestone: V2
       tests: []
+    }    {
+      name: back2back
+      desc: '''
+           randomly select the spacing between consecutive messages from 0 - n clock cycles'''
```

Comment:
Actually, "between consecutive blocks and messages".

Reply:
Please, also add a note that we are also going to measure performance/throughput with this test. Note: Performance is one of the classes in the STD_TEST_CATEGORIES_PLANNED item of the V1 checklist.

Reply:
I'd prefer it be a separate testplan item. It is ok to have multiple testplan entries pointing to the same set of tests.

Reply:
I would argue that all but performance and power is already in the testplan.

I added performance and I don't think power is applicable.

Stress seems out of place here.
since all the testpoints should do exactly that stress the DUT and push it into corners so isn't having it as a separate item redundant?

Reply:
Mildly agree - I put it in there to remind DV engineers to plan for a stress test. Each of your tests are likely targeting different features of the DUT and stressing it heavily.

But, we intend to go one step further in a stress test where we combine all those individual test sequences and run them together (sequentially or in parallel as applicable). We cross this with others things forked off (such as randomly injecting TL error packets and asserting reset randomly in the middle of the test). 

When auto-generating the testbench, the {aes_stress_all, aes_stress_all_with_rand_reset} tests are added automatically (you need to uncomment line 36 of your `aes_sim_cfg.hjson` file) - which does exactly the above.

Reply:
The intent of the testplan is to discretely capture everything that is intended to be verified. Written tests (the actual SV test / sequence class) can target more than one feature, but for the sake of clarity its better to capture them as separate items. It is hard for the reviewer to dig deep into the description and know that for instance, 'back2back' also covers 'performance' related features.